libraries/chain/resource_limits.cpp

```diff
@@ -344,59 +349,6 @@ int64_t resource_limits_manager::get_account_cpu_limit( const account_name& name
    int64_t cpu_weight;
    get_account_limits( name, unused, unused, cpu_weight );
```

Comment:
I would prefer if you used two different variables for the unused arguments (similar to the `x` and `y` used in `get_account_net_limit_ex`), so that we don't get confusing by spooky aliasing behavior while debugging `get_account_limits` in the future.